src/server/game/Spells/Auras/SpellAuras.h

```diff
@@ -138,6 +138,8 @@ class TC_GAME_API Aura
         void UpdateOwner(uint32 diff, WorldObject* owner);
         void Update(uint32 diff, Unit* caster);
 
+        virtual void OnTargetHeartbeat(AuraApplication* aurApp) { }
```

Comment:
```suggestion
        virtual void OnTargetHeartbeat(AuraApplication* /*aurApp*/) { }
```

Comment out the argument to silence compile warning